recipes-omnect/initrdscripts/omnect-os-initramfs/flash-mode-1

```diff
@@ -219,11 +215,11 @@ flash_mode_1_run() {
 
     run_flash_mode_1 2>&1 | tee /tmp/flash-mode-1.log >/dev/console 2>&1;
     return=${?}
-    blk_dev_data=$(get_block_device data)
+    blk_dev_data=$(readlink -f /dev/omnect/data)
     if [ -b "${blk_dev_data}" ]; then
         # store log file
         mkdir -p /tmp/mnt_data
-        check_ext4_fs "data" "${blk_dev_data}" 0
+        check_fs "${blk_dev_data}"
```

Comment:
does that really work?
readlink will return a path like /dev/mmcblk0p7, but check_fs expects a label now.
probably 'check_fs "data"' was meant instead?

Reply:
nope, thx. flash-mode-1 does work, but this not.
fixed.